src/renderer/Presentational/ContentMultipleClients/resources.json

```diff
@@ -0,0 +1,73 @@
+[
```

Comment:
We don't need to do now, but fyi we should have all node specific info, links, etc. in the respective /src/common/NodeSpecs/... file